common/scala/transformEnvironment.sh

```diff
@@ -43,6 +43,9 @@ fi
 if [ -n "$config" ]
 then
     location="/config.conf"
+    if [ -n "$NOT_ROOT_USER" ]; then
+        location="/home/$NOT_ROOT_USER/config.conf"
+    fi
```

Comment:
That makes sense to me. Should we always write to the current user's home directory to make the script as simple as possible?